ui/app/src/components/workspaces/WorkspaceItem.tsx

```diff
@@ -48,9 +49,11 @@ export const WorkspaceItem: React.FunctionComponent<WorkspaceItemProps> = (props
         </PivotItem>
         <PivotItem headerText="History">
           <h3>--History goes here--</h3>
+
+
         </PivotItem>
         <PivotItem headerText="Operations">
-          <h3>--Operations Log here</h3>
+            <WorkspaceOperationsPanel workspace={props.workspace} />
```

Comment:
I think since we're listing Operations, we could change this to `ResourceOperationsList` rather than it being specific to a workspace - then we could re-use it for every resource type.

Reply:
Agree. Thanks!